.editorconfig

```diff
@@ -10,6 +10,7 @@ root = true
 insert_final_newline = true
 indent_style = space
 indent_size = 4
```

Comment:
If a forgotten file extension isn't included in .editorconfig, it may use indent_size = 4 by mistake. I think it makes more sense to move this option to .cs and .vb files only.